Modules/socketmodule.c

```diff
@@ -5574,6 +5574,23 @@ static PyType_Spec sock_spec = {
     .slots = sock_slots,
 };
 
+static PyObject *
+socket_getattr(PyObject *self, PyObject *name)
+{
+    const char *attrname = PyUnicode_AsUTF8(name);
+    PyObject *sock_type = PyType_FromSpec(&sock_spec);
+    if (sock_type == NULL) {
+        return NULL;
+    }
+
+    if (!strcmp(attrname, "SocketType")) {
+        PyErr_Warn(PyExc_DeprecationWarning, "Use socket.socket type instead");
+        return sock_type;
+    }
+
+    PyErr_Format(PyExc_AttributeError, "module _socket has no attribute %s", attrname);
```

Comment:
```suggestion
    PyErr_Format(PyExc_AttributeError, "module _socket has no attribute '%s'", attrname);
```

In order to match the Python implementation message.